### ISSUE TITLE
error occurred at dialog prompt

### ISSUE THREAD

**Issue by UrVerySpecial** [rafaelpimpa/buefy#54]
i got error when submitted at dialog prompt
```
Uncaught TypeError: Cannot read property 'checkValidity' of undefined
    at o.html5Validation (app.ed77701….js:1)
    at o.n [as html5Validation] (vendor.726eec4….js:6)
    at o.blur (app.ed77701….js:1)
    at n (vendor.726eec4….js:6)
    at HTMLInputElement.t (vendor.726eec4….js:6)
    at Object.Ot [as removeChild] (vendor.726eec4….js:6)
    at r (vendor.726eec4….js:6)
    at n (vendor.726eec4….js:6)
    at vendor.726eec4….js:6
    at vendor.726eec4….js:6
    at c (vendor.726eec4….js:6)
    at HTMLDivElement.d (vendor.726eec4….js:6)
```
![error](https://cloud.githubusercontent.com/assets/5692265/25832807/e6ade372-34a7-11e7-9ce6-a2af7da28d6c.gif)

**Comment by rafaelpimpa** [comment]
Gonna take a look at it tomorrow, did you clone or is it in the online docs?

**Comment by UrVerySpecial** [comment]
in online docs site!

**Comment by rafaelpimpa** [comment]
This is not happening here 🤔 
Tested in Chrome, Firefox and Edge.

**Comment by UrVerySpecial** [comment]
😟 I got this error in Chrome. And my team's browser too. I've tested at firefox, and it was OK.
 I have no clue why it is ok at your env...
I figure it is not critical problem, so there is no problem to close this issue.
Thanks.

**Comment by rafaelpimpa** [comment]
It might've been fixed at the v0.3.2, could you take a look?
